### DISCUSSION TITLE
输出后有白色异常色块

### DISCUSSION THREAD

**Discussion by pualmac** [Tencent/libpag#1827] [Q&A]
如演示所示，卡片在进行旋转前有几帧白色与卡片大小相等的色块，但在AE工程内没有发现有其他图层或其他素材
[Uploading 领取成功.zip…]()

**Comment by kevingpqi123** [comment]
此类问题建议提供 AE 工程文件，首先对比 AE 中效果和导出 pag 文件后桌面端的预览效果，看是否是一致的，然后根据结果做进一步的排查
